src/components/ToDos/ToDoList/index.js

```diff
@@ -19,6 +19,8 @@ export default function ToDoList() {
 				return toDos.filter(todo => todo.completed === true);
 			case 'uncompleted':
 				return toDos.filter(todo => todo.completed === false);
+			case 'urgent':
+				return toDos.filter(todo => todo.urgent === true && todo.completed === false);
```

Comment:
Actually, you don't need  `todo.urgent` is truthy and `!todo.completed` is falsy - but that is just sugar 🍭
```suggestion
				return toDos.filter(todo => todo.urgent && !todo.completed);
```